devel/update_all

```diff
@@ -0,0 +1,19 @@
+#! /bin/bash
+# update and rebuild from source
+set -e
+set -x
+cd istio
+git pull
+cd ../mixer
+git pull
+cd ../proxy
+git pull
+cd ../wrk
+git pull
+make
+cd ../proxy
+bazel build src/envoy/mixer:envoy
+cd ../mixer
+bazel build ...
```

Comment:
For Mixer, you could build

cmd/server

And that'll produce mixs and nothing else.